lighty-netconf-device/src/main/java/io/lighty/netconf/device/requests/OkOutputRequestProcessor.java

```diff
@@ -41,8 +40,7 @@
     }
 
     @Override
-    protected String convertNormalizedNodeToXmlString(NormalizedNode normalizedNode)
-            throws SerializationException {
+    protected String convertNormalizedNodeToXmlString(NormalizedNode normalizedNode) {
```

Comment:
no more we throws exception?

Reply:
Reverted and fixed import for SerializationException.